### PR TITLE
Added IsZombie(), IsHuman(), rewrote IsInfected() to IsZombie()

### PR DESCRIPTION
**Pull request by teoman002**
New function: IsZombie()
New function: IsHuman()

### CODE REVIEW

File: src/game/server/gamemodes/mod.cpp [hunk 1]
```diff
@@ -553,12 +553,12 @@ int CGameControllerMOD::OnCharacterDeath(class CCharacter *pVictim, class CPlaye
...
 		
-	if(pKiller->IsInfected())
+	if(pKiller->IsZombie())
 	{
 		CPlayer* pVictimPlayer = pVictim->GetPlayer();
 		if(pVictimPlayer)
 		{
-			if(!pVictim->IsInfected())
+			if(!pVictim->IsZombie())
 			{
```
Comment (duralakun) [inline src/game/server/gamemodes/mod.cpp]: could be IsHuman but is no mistake

**Comment by teoman002** [comment]
@duralakun I fixed the lines you mentioned!

File: src/game/server/entities/character.cpp [hunk 2]
```diff
@@ -2826,12 +2826,12 @@ bool CCharacter::TakeDamage(vec2 Force, int Dmg, int From, int Weapon, int Mode)
 		else
 		{
 			//If the player is a new infected, don't infected other -> nobody knows that he is infected.
-			if(!pKillerPlayer->IsInfected() || (Server()->Tick() - pKillerPlayer->m_InfectionTick)*Server()->TickSpeed() < 0.5) return false;
+			if(!pKillerPlayer->IsZombie() || (Server()->Tick() - pKillerPlayer->m_InfectionTick)*Server()->TickSpeed() < 0.5) return false;
```
Comment (duralakun) [inline src/game/server/entities/character.cpp]: IsHuman() - but its not so important

File: src/game/server/entities/character.cpp [hunk 3]
```diff
@@ -2976,7 +2976,7 @@ void CCharacter::Snap(int SnappingClient)
 /* INFECTION MODIFICATION START ***************************************/
 	if(GetClass() == PLAYERCLASS_GHOST)
 	{
-		if(!pClient->IsInfected() && m_IsInvisible) return;
+		if(!pClient->IsZombie() && m_IsInvisible) return;
```
Comment (duralakun) [inline src/game/server/entities/character.cpp]: IsHuman

**Pull request merged**